modules/downloading-manifest-files-resources-oci.adoc

```diff
@@ -0,0 +1,48 @@
+
+.Prerequisites
+
+* You uploaded a generated Agent ISO image to {oci}. See the "Using the Assisted Installer to generate an OCI-compatible Agent ISO image" section.
+* You have permissions to access the `oracle-quickstart / oci-openshift` GitHub repository.
+
+.Procedure
+
+. From the link:https://github.com/oracle-quickstart/oci-openshift[`oracle-quickstart / oci-openshift`] GitHub web page, click the **<> Code** button and then click **Download ZIP**.  The following list details these resources:
++
+** link:https://github.com/oracle-quickstart/oci-openshift/tree/main/custom_manifests[CCM and CSI custom manifests].
+** link:https://github.com/oracle-quickstart/oci-openshift/blob/main/README.md[Download the {oci} Resource Manager configuration to provision resources for deploying Openshift on {oci}].
++
+[NOTE]
+====
+Consider using the example configurations in link:https://github.com/oracle-quickstart/oci-openshift/tree/main/custom_manifests/manifests[manifests] GitHub page as a guide for configuring a cloud infrastructure to meet your organization's needs.
+====
+
+. Go to **Resource Manager > Stacks** and click the **Create stack** button, and upload the archive file, `.zip`, that you downloaded previously from the `oracle-quickstart / oci-openshift` repository.  
+
+. Complete the following actions to create a stack:
++
+.. From the **Stack information** section, specify a name for your stack.
+.. From the **Configure variables** section, complete the following fields. Ensure you replace the examples with your actual values:
+... **cluster_name:**  `ocicluster`.
+... **compartment_ocid:** Specify the {ocid} from the parent compartment. For example, `ocid1.compartment.oc1..aaaaaaaa6r2iu3qndqgz5ogqkgnh2u2ajy5iou5ugkjr2ksmkrtdqrvxsvyq`.
+... **home_region:**  `us-sanjose-1`
+... **zone_dns:**  `openshift-demo.devcluster.openshift.com`.
+... **enable_private_dns:** Set the DNS zone to public or private. Specifying a value of `true`, creates a private DNS zone; specifying a value of `false`, creates a public DNS zone. For a private DNS zone,  you must configure your local `/etc/hosts` file to reach the cluster.
+... **openshift_image_source_url:** Specify the URL you copied from a previous step of the quick-start procedure.
+.. Click the **Next** button.
+.. From the **Review** section, review your details and then click the **Create** button. 
+
+. Click the **Apply** button to start an apply job. Check the **Logs** section to confirm your stack was successfully created. 
+
+. Copy the `oci_ccm_config` cloud configuration from the **Outputs** section of the log. 
+
+. Extract the archive file that you downloaded in a previous step from the link:https://github.com/oracle-quickstart/oci-openshift[`oracle-quickstart / oci-openshift`] repository. A new directory displays on your file explorer. Open the directory in your preferred code editor.
```

Comment:
@faermanj . The following sentence in this step just floats a bit. Can I add something more to explain why a customer should open the directory in their preferred editor? Is it best to drop this sentence?

> A new directory displays on your file explorer. Open the directory in your preferred code editor.

Reply:
Why do we need to extract files in this step? What are we going to do ? 
I think it shoudl be moved to next steps where we edit oci-ccm.yml and oci-csi.yml.
Or better just remove from here.